ironfish-cli/src/commands/reset.ts

```diff
@@ -45,6 +45,8 @@ export default class Reset extends IronfishCommand {
       this.exit(0)
     }
 
+    this.sdk.config.set('networkId', this.sdk.config.defaults.networkId)
+    await this.sdk.config.save()
```

Comment:
I think these need to be `this.sdk.internal`

We may want to set `isFirstRun` to true, too: since we're deleting the wallet database the node will start up without creating an account unless it's the first run.